src/main.rs

```diff
@@ -672,7 +672,26 @@ fn exec_cargo_inner(
     if progress.count != 0 && !cx.print_command_list && cx.log_group == LogGroup::None {
         eprintln!();
     }
-    progress.count += 1;
+
+    let new_count = progress.count + 1;
+    let mut skip = false;
+    if let Some(partition) = &cx.partition {
+        if progress.count % partition.count != partition.index - 1 {
+            let mut msg = String::new();
+            if term::verbose() {
+                write!(msg, "skipping {line}").unwrap();
+            } else {
+                write!(msg, "skipping {line} on {}", cx.packages(id).name).unwrap();
+            }
+            write!(msg, " ({}/{})", new_count, progress.total).unwrap();
+            let _guard = cx.log_group.print(&msg);
+            skip = true;
+        }
+    }
+    progress.count = new_count;
+    if skip {
+        return Ok(());
+    }
 
     if cx.clean_per_run {
```

Comment:
(unrelated to this pr: i don't think `cargo_clean(...)` is needed if `cx.print_command_list == true`..right?)

Reply:
Yeah. Well, both are optional and I don't know what the use case of using --clean-per-run together with -pr-int-command-list in the first place, though...

Reply:
thanks for quick reply. I'll create a small pr after this pr to fix this.